templates/arm-aks-template.json

```
@@ -123,7 +127,7 @@
     {
       "type": "Microsoft.ContainerService/managedClusters",
```

doesn't it have to have be "type": "Microsoft.ContainerService/EnableEphemeralOSDiskPreview"
```suggestion
      "type": "Microsoft.ContainerService/managedClusters",
```